modules/translators/base.py

```diff
@@ -39,6 +39,7 @@
     'Türk dili': '',
     'украї́нська мо́ва': '',  
     'Thai': '',
+    'Arabic': 'ar',
```

Comment:
Please change it to ```'Arabic': '',```

Reply:
Sorry. I thought I changed it when I made the change, but apparently I missed it. Thanks)